package.json

```diff
@@ -91,6 +91,7 @@
     "@playwright/test": "1.37.1",
     "@qlik/eslint-config": "0.4.17",
     "@qlik/prettier-config": "0.3.0",
+    "@testing-library/react": "^14.0.0",
```

Comment:
The versions should be pinned. Also don't forget to run `yarn` otherwise the lock file won't have pinned versions.